presto-main/src/main/java/com/facebook/presto/operator/aggregation/noisyaggregation/ScaleRandomSeedState.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.noisyaggregation;
+
+import com.facebook.presto.spi.function.AccumulatorState;
+
+public interface ScaleRandomSeedState
```

Comment:
what's the value of having this interface separate from CountScaleRandomSeedState which extends it.  I don't see where it's used anywhere else, so why not have a single interface?

Reply:
They are there for other noisy aggregations which also have options for noiseScale and randomSeed. E.g., there is another PR coming for noisy_sum_aggregation. I can put them into 1 interface now and do other refactoring in the next PR for other aggregations